features/goscripts/cilium.go

```diff
@@ -0,0 +1,153 @@
+package features
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+func (test *TestRun) IRunVARInVARDirectory(arg1, arg2 string) error {
+	arg1 = test.VarMap[arg1]
+	err := test.IRunInVARDirectory(arg1, arg2)
+	return err
+}
+
+func (test *TestRun) IRunInVARDirectory(arg1, arg2 string) error {
+	arg2 = test.VarMap[arg2]
+	err := test.IRunInDirectory(arg1, arg2)
+	return err
+}
+
+func (test *TestRun) VARIABLEEqualsContainersFROMOutput(arg1, arg2 string) error {
+	var err error
+	tmp := strings.Split(fmt.Sprintf("%s", string(test.Output)), "\n")
+	for index, elem := range tmp {
+		if strings.Contains(elem, arg2) && !strings.Contains(elem, "operator") { //--- "operator" is to exclude cilium-operator
```

Comment:
we can't hardcode this in the function, we need to parameterize this to have something like "variable containers $VAR1 and not $VAR2"

Reply:
Did you see the function???
 VARIABLEEqualsContainersFROMOutput
That's a function that I will use exclusively for cilium. Hardcoded. That's why it stays in cilium.go.

Reply:
but can't we make it generic ? it seems that something we can reuse elsewhere

Reply:
Does that have to be resolved _now_ or can we mark it with a `#FIXME`?
This PR is somewhat huge, I would prefer to merge it with some requested changes, and open another PR who focus on refactoring.

Reply:
yes sure, it's not blocking at all ;)

Reply:
Sure, you can make it generic if you want.